conda-forge.yml

```
@@ -1,6 +1,9 @@
+conda_build:
```

Maybe also try adding the OSX ARM64 migration, while we're here:

```suggestion
build_platform:
  osx_arm64: osx_64
conda_build:
```